lib/flow.js

```diff
@@ -47,7 +47,8 @@ const reportDir = function reportDir(cb) {
     log('reportDir: output:reports not defined');
     return cb(null, this);
   }
-  let tsDirName = moment().format('MM-DD-YYYY/HH-mm-ss');
+  const pathTimestampFormat = this.config.get('output:pathTimestampFormat') || 'MM-DD-YYYY/HH-mm-ss';
+  let tsDirName = moment().format(pathTimestampFormat);
```

Comment:
Since this is a user supplied configuration, we may want to wrap this in a try/catch or otherwise figure out how to handle the case where the value provided is invalid.

Reply:
I'm also interested in what issues arise if we pass in a "valid" but unexpected value like `foo`.  Will we run into errors if multiple tests/runs try to write to the same folder?

Perhaps we don't have to handle all these cases and users will be able determine the errors are due to their custom config value 🤷‍♂️